### PR TITLE
Cgroup resource limits

### PR DESCRIPTION
**Pull request by Werkov**
### Description
Users may encounter former limits not applying to new environments, hence add explanations about limit setting. Also few minor fixes when being around.

### Checklist
* Check all items that apply.

*Are backports required?*

- [x] To maintenance/SLE15SP1
- [ ] To maintenance/SLE15SP0
- [x] To maintenance/SLE12SP5
- [x] To maintenance/SLE12SP4
- [ ] To maintenance/SLE12SP3

### CODE REVIEW

File: xml/tuning_cgroups.xml [hunk 1]
```diff
@@ -66,6 +66,12 @@
    slice, scope, socket, mount, or swap name. Replace properties with
    one or more of the following:
   </para>
+  <para>
```
Comment (cjschroder) [inline xml/tuning_cgroups.xml]: This is a bit awkward. I suggest moving the paragraph "Limitations to <literal>cgroups</literal>..." after "Beware that resource consumption...", and make "Beware that resource consumption..." a Note, like this:


&lt;note>
&lt;title>Resource Consumption Depends on Workload&lt;/title>
&lt;para>
...
&lt;/para>
&lt;/note>

Comment (cjschroder) [reply xml/tuning_cgroups.xml]: I am happy to make the backports, if you like.

Comment (Werkov) [reply xml/tuning_cgroups.xml]: Thanks for the review. I've force-pushed your proposals. Feel free to edit it as necessary for the common document formatting, I don't care that much as long as it conveys the message.

**Comment by cjschroder** [comment]
Thanks @Werkov, much appreciated!

**Comment by cjschroder** [comment]
@Werkov does this also apply to master (SLE15SP2) ?

**Comment by Werkov** [comment]
Yes, it remains valid even in SLE15 SP2.

**Comment by cjschroder** [comment]
:+1: Thanks!

**Comment by cjschroder** [comment]
sle12sp5: https://github.com/SUSE/doc-sle/commit/3eca7b08d9e0a1805eeb38fe2878c8c008f2dfa3
sle12sp4: https://github.com/SUSE/doc-sle/commit/8f7d6629d4e7f12692ddfe1c49e65cdc22f99a84
master (sle15sp2): https://github.com/SUSE/doc-sle/commit/99acb31a49eba161607182345bfe5627cd60dddc

**Pull request merged**